### ISSUE TITLE
Support for TEMPer1F_H1_V1.4 devices?

### ISSUE THREAD

**Issue by nmcoliver** [rcrum003/TEMPered-v6-2015#3]
Hi Ryan, did you manage to include the support for TEMPer1F_H1_V1.4 or TEMPer1F_H1_V1.5 devices? Mine is reported as TEMPer1F_H1_V1.4 with "hid-query -e", and TEMPer1F_H1V1.5F with "tempered". Thanks.

**Comment by rcrum003** [comment]
Sounds like a new variant! Give me a couple of days, and I’ll see if i can add support for it.

I may need you to run a few commands to get me the right strings.

Do you get error when you run the current version?

Ryan

**Comment by nmcoliver** [comment]
Yep, looks like a new device. Detection is even problematic. With the current version I get:

$ hid-query -e
/dev/hidraw3 : 0c45:7402 interface 0 : RDing TEMPer1F_H1_V1.4
/dev/hidraw4 : 0c45:7402 interface 1 : RDing TEMPer1F_H1_V1.4

$ tempered 
/dev/hidraw4: Could not open device: Unknown device subtype string: TEMPer1F_H1V1.5F

so even the version detection is incoherent. Adding a new device type, copying the TEMPerHumM12V1.3 definitions in libtempered/temper_type.c I get:

$ ./tempered 
/dev/hidraw4 0: temperature -28.58 °C, relative humidity -2.1%, dew point nan °C

so maybe a different sensor? That's why I wondered if you managed to get reasonable values out of any TEMPer1F_H1_V1.4 devices, at least.

**Comment by rcrum003** [comment]
The subtype error is because we dont handle that subtype.

This will help me fix the code. I'll repost when it's ready to test.
